### PR TITLE
Markdown linting, capitalization consistency, wording updates

### PR DESCRIPTION
**Pull request by krook**
Signed-off-by: Daniel Krook <krook@us.ibm.com>

These changes apply to Markdown, Python sample code, and Java sample code. I'll open a separate pull request for the typo fixes in the notebooks themselves.

* Markdown linting
* Python linting with Black
* Java formatting with Red Hat Java Plugin for VSCode
* Checked links
* Fixed typos
* Added link to form to get trial keys needed for container library download

### CODE REVIEW

**Review by chen-chris-topher — COMMENTED** [review]
[MLOps/Watson-NLP-Container/readme.md](https://github.com/ibm-build-lab/Watson-NLP/pull/5/files#diff-5cbe22288859da4168523a4b9146dd37e1c00eb55158c9bac22300ab9828ee7a) lines 122-160 showing up with red highlighting...is this intended?

**Comment by krook** [comment]
> [MLOps/Watson-NLP-Container/readme.md](https://github.com/ibm-build-lab/Watson-NLP/pull/5/files#diff-5cbe22288859da4168523a4b9146dd37e1c00eb55158c9bac22300ab9828ee7a) lines 122-160 showing up with red highlighting...is this intended?

Good eye. The code output snippet isn't valid JSON as I hinted in the block. I set it back to something that doesn't validate syntax (`sh` rather than `json`) so that it doesn't flag it as broken. The red text is gone. And since it's not real code it's not going to break a build or anything. 

Preview here:
https://github.com/ibm-build-lab/Watson-NLP/tree/text/nov2/MLOps/Watson-NLP-Container#4-test

**Comment by krook** [comment]
@chen-chris-topher @mjspriggs @htalukderibm @kevinxhuang 

Anyone have some time to review?

**Review by mjspriggs — CHANGES_REQUESTED** [review]
In Init-Container/client/GrpcClient.py, this is going to break something on line 13.  Shouldn't add a comma.

Same on line 30:

        SENTIMENT_DOCUMENT_CNN_WORKFLOW_MODEL = os.getenv(
            "SENTIMENT_DOCUMENT_CNN_WORKFLOW_MODEL",
            default="sentiment_aggregated-cnn-workflow_lang_en_stock",
        )
        
And line 44:

        EMOTION_CLASSIFICATION_STOCK_MODEL = os.getenv(
            "EMOTION_CLASSIFICATION_STOCK_MODEL",
            default="classification_ensemble-workflow_lang_en_tone-stock",
        )

File: Getting-Started.md [hunk 1]
```diff
@@ -1,55 +1,54 @@
+# Setting up your environment
```
Comment (Shivam6693) [inline Getting-Started.md:1]: This file has been removed as the tutorial has been published on IBM developer

Comment (krook) [reply Getting-Started.md:1]: There might be some legacy links to these files out there, can we instead replace the superseded markdown files with links to their new location on IBM Developer?

**Comment by krook** [comment]
Thanks for the review @mjspriggs. Do the trailing commas on the last item in a list actually break the code? I noticed that the Python Black linter added them on imports too. It didn't seem to cause an issue in my env. Can you confirm?

File: Getting-Started.md (hunk 1 above)
Comment (krook) [reply Getting-Started.md:1]: For example, that's what this file does in the Speech repo: https://github.com/ibm-build-lab/Watson-Speech/blob/main/Speech%20To%20%20Text/Speech-To-Text-Tutorial.md

Comment (krook) [reply Getting-Started.md]: Merged my branch with your updates to main to include these changes.

**Comment by krook** [comment]
> > In Init-Container/client/GrpcClient.py, this is going to break something on line 13. Shouldn't add a comma.

> Thanks for the review @mjspriggs. Do the trailing commas on the last item in a list actually break the code? I noticed that the Python Black linter added them on imports too. It didn't seem to cause an issue in my env. Can you confirm?

Ok, I did confirm that the trailing comma shouldn't be a problem. It's also something that's generally recommended by the PEP8 and Black standards as well. 

https://peps.python.org/pep-0008/#when-to-use-trailing-commas
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#trailing-commas

**Comment by mjspriggs** [comment]
> Ok, I did confirm that the trailing comma shouldn't be a problem. It's also something that's generally recommended by the PEP8 and Black standards as well.

@krook OK, thanks for pointing that out.

**Review by mjspriggs — APPROVED** [review]

**Comment by mjspriggs** [comment]
@krook Please wait for all reviewers to approve before committing.  We have partitioned the review process across the team.

The review assignments are here:   https://ibm.ent.box.com/notes/1048442856096

**Review by himadrica — APPROVED** [review]

**Review by Shivam6693 — APPROVED** [review]

**Review by htalukderibm — APPROVED** [review]

**Review by Abhilasha-Mangal — APPROVED** [review]

**Review by Abhilasha-Mangal — COMMENTED** [review]
readme.md  file is missing

**Comment by krook** [comment]
Looks like all reviews are done and approved. Ready to merge.

**Pull request merged**